lib-api/src/main/java/org/ergoplatform/appkit/InputBox.java

```diff
@@ -50,6 +50,15 @@
      */
     InputBox withContextVars(ContextVar... variables);
 
+    /**
+     * Json representation of this transaction.
+     */
     String toJson(boolean prettyPrint);
+
+    /**
+     * Json representation of this transaction.
+     */
+    String toJson(boolean prettyPrint, boolean formatJson);
```

Comment:
@wclaeys please add JavaDoc describing logic of the parameters and how they affect the output.